sdk-facebook/src/main/java/com/reach5/identity/sdk/facebook/FacebookProvider.kt

```diff
@@ -93,7 +94,12 @@ class ConfiguredFacebookProvider(
 
             override fun onError(error: FacebookException?) {
                 if (error != null) {
-                    failure(ReachFiveError.from(error))
+                    if (error is FacebookAuthorizationException) {
```

Comment:
Is this change related to webauthn ?

Reply:
Nope, I had an issue with my Facebook session and I couldn't logout. With this change, if there is a Facebook error, the user is automatically logout.